### PR TITLE
Users model testing

### LINKED ISSUE #85: write user model tests

### PR DESCRIPTION
**Pull request by leranf**
#### What's this PR do?

implements tests for user model
#### What are the relevant tickets/issues?

closes #85 
#### Where should the reviewer start?

/**tests**
#### Are there tests written yet? How should this be manually tested?

mocha <pathToUserTest>
#### Questions:
- Does this add new dependencies?
  request-promise
- Does our documentation need to be updated?
  no

### CODE REVIEW

File: server/__tests__/users-test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  it('should retrieve a user\s friends from the database', () => {
+    const friendIds = ['ad7-lr0-fd8', 'of8-a6s-lf0', 'qw0-lm1-pt8'];
+    const profile = {
+      displayName: testUsers[0].name,
+      photos: [{ value: testUsers[0].picUrl }],
+      id: testUsers[0].fbId,
+    };
+
+    User.findOne({ where: { name: 'Akshay Buddiga' }}).then(user => {
+      Promise.all(friendIds.map(fbId => User.findOne({ where: { fbId }})))
```
Comment (xuejinglu) [inline server/__tests__/users-test.js:71]: :100:  :+1:

File: server/__tests__/users-test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  it('should update a user in the database', () => {
+    const profile = {
+      displayName: testUsers[1].name,
+      photos: [{ value: testUsers[1].picUrl }],
+      id: testUsers[1].fbId,
+    };
+
+    // checks that it creates a new user
+    User.findOrCreate(profile).then(user => {
```
Comment (abuddiga) [inline server/__tests__/users-test.js:54]: :100:

File: server/__tests__/users-test.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    const profile = {
+      displayName: testUsers[0].name,
+      photos: [{ value: testUsers[0].picUrl }],
+      id: testUsers[0].fbId,
+    };
+
+    User.findOne({ where: { name: 'Akshay Buddiga' }}).then(user => {
+      Promise.all(friendIds.map(fbId => User.findOne({ where: { fbId }})))
+        .then(friends => 
+          user.addFriends(friends).then(() =>
```
Comment (abuddiga) [inline server/__tests__/users-test.js:73]: :+1:

**Comment by boyajay** [comment]
sveeeeeeet

**Pull request merged**